live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/transform/response/ATSTransformResponseHandler.java

```diff
@@ -51,33 +55,66 @@ public class ATSTransformResponseHandler extends RouteBuilder
 
     private final IngestContentCommandHandler ingestContentCommandHandler;
     private final IntegrationProperties integrationProperties;
+    private final ATSTransformRequester atsTransformRequester;
 
     @Override
     public void configure()
     {
-        // @formatter:off
+        onException(EmptyRenditionException.class, ResourceNotFoundException.class)
+                .process(this::retryContentIngestion);
+
+        onException(Exception.class)
```

Comment:
Is retry triggered in case of every exception? This is expected?

Reply:
hah, there are 2 tickets and one says ([ACS-6474](https://hyland.atlassian.net/browse/ACS-6474)) that we should retry on any IOException (which is never thrown as it checked exception so I broaden it to just any Excpetion) and there is second ticket ([ACS-6475](https://hyland.atlassian.net/browse/ACS-6475)) that says that we shouldn't retry on any Exception😂 I would leave it as it is and create new ticket that will detail what should we handle in which way

Reply:
This sounds reasonable 👍